covid19-dashboard/src/DataTable.tsx

```diff
@@ -195,33 +184,24 @@ export default class DataTable extends React.Component<
           <Th timeSeries={timeSeries}
             typeOf={category.typeOf}
             key={index}
-            className={clickableClass}
             graphTitle={place.name}
             graphSubtitle={category.graphSubtitle}
-            color={category.color}
-          />
+            color={category.color}/>
         );
       });
 
       const subregionType = place.getSubregionType()
 
-      let placeFullName: string;
+      let placeFullName = place.name
       if (place.placeType !== 'Country') {
-        placeFullName = `${place.name}, ${place.parentPlace?.name}`;
-      } else {
-        placeFullName = place.name;
-      }
-
-      let onClick = () => {};
-      if (subregionType) {
-        onClick = () => goToPlace(place.geoId, subregionType)
+        placeFullName += `, ${place.parentPlace?.name}`;
       }
 
       return (
         <tbody key={index}>
-          <tr key={index}
-              className={clickableClass}
-              onClick={onClick}>
+          <tr className={subregionType ? 'clickable' : ''}
+              {...(subregionType && {
```

Comment:
Is this what you meant by "only register onClick if"

Reply:
if there is a subregion, then onClick is registered, otherwise no onClick is passed.

Reply:
yes, this is what i meant. any css changes to add here? updated color or font decoration, cursor: pointer, etc to denote that something is clickable vs other non-clickable things?

Reply:
It was added along the HTML file. There is clickable:hover in the index.css file.

https://github.com/datacommonsorg/tools/pull/37